oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -28,6 +28,8 @@ cluster:
     enableACL: ${SW_ZK_ENABLE_ACL:false} # disable ACL in default
     schema: ${SW_ZK_SCHEMA:digest} # only support digest schema
     expression: ${SW_ZK_EXPRESSION:skywalking:skywalking}
+    internalComHost: ${SW_CLUSTER_INTERNAL_COM_HOST:""}
+    internalComPort: ${SW_CLUSTER_INTERNAL_COM_PORT:-1}
```

Comment:
Please update these two in other coordinator configurations with docs.

Reply:
done